ci/requirements-openvino-dev.in

```diff
@@ -0,0 +1,30 @@
+addict
```

Comment:
is it possible to get this list from wheel in requirements updating script instead of hardcoding here?

Reply:
e.g. using this solution or do something similar
https://pypi.org/project/pkginfo/

Reply:
Done.